stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/requests/RemoveMembersRequest.kt

```diff
@@ -0,0 +1,8 @@
+package io.getstream.chat.android.client.api2.model.requests
+
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+internal data class RemoveMembersRequest(
+    val remove_members: List<String>,
```

Comment:
Does this work with obfuscation?

Reply:
Moshi's generated adapters include the field names as strings, plus it generates ProGuard rules which should prevent this class from being obfuscated.